.changeset/modern-books-compare.md

```diff
@@ -68,8 +68,9 @@
 '@twilio-paste/customization': major
 '@twilio-paste/icons': major
 '@twilio-paste/style-props': major
+'@twilio-paste/core': minor
```

Comment:
I'm just improving the changeset for the peer depenedencies here. I believe it will prevent the major version on core.